askama_derive/src/generator.rs

```diff
@@ -226,13 +247,13 @@ impl<'a> Generator<'a> {
                 #where_clause
             )
         ))?;
-        buf.writeln("type Error = ::askama::Error;")?;
+        buf.writeln("type Error = ::askama_hyper::askama::Error;")?;
```

Comment:
I think all instances of `::askama_something::askama::` have to be replaced with `::askama_something::` now that we don't export a symbol `askama` in the integration crates.